src/items.ts

```diff
@@ -1,124 +1,1166 @@
+/**
+ * items.ts
+ * All available items and their properties
+ * lgfrbcsgo & Nikolaus - October 2020
+ */
+
 export type Liter = number
 export type Minutes = number
 export type Quantity = number
 
-export type Item = Machine | Container | Ore | Other
+export type Item = Ore | Catalyst | Product | Element
 
 export enum ItemType {
-    MACHINE = "machine",
-    CONTAINER = "container",
     ORE = "ore",
-    OTHER = "product",
+    CATALYST = "catalyst",
+    PRODUCT = "product",
+    ELEMENT = "element",
 }
 
-export interface Container extends CommonProperties {
-    readonly type: ItemType.CONTAINER
+/**
+ * Properties commen to each item type
+ */
+export interface CommonProperties {
+    readonly name: string
+    readonly tier: string
+    readonly category: string
```

Comment:
`tier` and `category` should probably be an enum.

Reply:
Done in https://github.com/lgfrbcsgo/du-factory-generator/commit/b2ca5abf70d5f5d1f8272961ae611d9e8b5b9bf3